security-admin/src/main/resources/conf.dist/ranger-admin-default-site.xml

```diff
@@ -681,4 +681,8 @@
 		<name>ranger.allow.kerberos.auth.login.browser</name>
 		<value>false</value>
 	</property>
+	<property>
+		<name>ranger.policy.retriever.transaction.timeout</name>
```

Comment:
What's the time unit? Could you please follow the existing configuration key naming style?
`<name>ranger.resource.lookup.timeout.value.in.ms</name>`